File: BACK-END/KuentecoApp/src/main/java/org/kuenteco/backend/dto/image/ImageDTO.java
package org.kuenteco.backend.dto.image;

import jakarta.validation.constraints.NotBlank;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class ImageDTO {
    @NotBlank private String name;
    @NotBlank private String imageUrl;
    private String imageId;
}

File: FRONT-END/lib/dto/auth/response/image_dto.dart
class ImageDTO {
  final String name;
  final String imageUrl;
  final String imageId;

  ImageDTO({required this.name, required this.imageUrl, required this.imageId});

  factory ImageDTO.fromJson(Map<String, dynamic> json) {
    return ImageDTO(
      name: json['name'],
      imageUrl: json['imageUrl'],
      imageId: json['imageId'],
    );
  }
}